core/editor/Cargo.toml

```diff
@@ -10,3 +10,8 @@ license = "Apache-2.0"
 
 [dependencies]
 log = "0.4"
+bitflags = "1.2.1"
+
+[dependencies.document-core]
+path = "../document"
+package = "graphite-document-core"
```

Comment:
This got included from the other commit, if it matters.

Reply:
Doesn't do too much harm, so I think I'll just leave it there